app/controllers/application_controller.rb

```diff
@@ -1,2 +1,10 @@
 class ApplicationController < ActionController::API
+  before_action :authenticate_user!
+  before_action :configure_permitted_parameters, if: :devise_controller?
+
+  protected
+
+  def configure_permitted_parameters
+    devise_parameter_sanitizer.permit(:sign_up, keys: %i[email username password password_confirmation])
+  end
```

Comment:
Hello Medson, users don't need to sign up, they need to log in. I hope you should add: `permit(sign_in, keys: %i[username])`